content/events/Berlin-blockchain-week-depin-day.md

```diff
@@ -0,0 +1,45 @@
+---
+eventName: "Berlin Blockchain Week - DePIN Day" #the name of your track or event and its mandatory
+
+eventDescription: "Dive into the latest topics and research in DePin and decentralized compute including multi-party computation, decentralized RPC, capacity rewards and proof systems." #short description of your track or event limiting to 100-150 characters
+
+#Topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some references for topics - Blockchain, Web3, Cryptocurrency, Tech Talks, Workshop, etc.
+
+eventTopic: 
+  - "DePIN"
+  - "Compute"
+  - "<Topic_3>"
+  - "<Topic_4>" 
+
+eventType: "Conference" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Berlin, Germany"  #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+venueName: "" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "Register to see address" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "" #the event venue Map link (will show up on a map) or just leave it blank
+
+website: "https://lu.ma/depin-berlin" #make sure to have all the relevant information: dates, venue, program, ticketing (if any), etc. or just leave it blank
+
+startDate: "05/22/2024" # the start date of the event , date format is MM/DD/YYYY eg: if it is February 16th 2023 => 02/16/2023
+
+endDate: "05/22/2024" #the end date of the event , date format is MM/DD/YYYY eg: if it is February 18th 2023 => 02/18/2023
+
+tag: "Industry Event" #Mention any one of the event types. Don't leave it blank.
+
+dateTBD: false #If you don't have concrete dates, add the approximate dates & set dateTBD: true.
+
+#Preferred means of contacts - this is a placeholder for email(for eg:  - email|mailto:<email_id>) and other social handles like Twitter, LinkedIn, Discord, etc. (for eg.   - 'twitter|https://twitter.com/IPFS/status/1629199396700098560?s=20')
+
+preferredContacts:
+  - '<contact_means>|<contact_url>'
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - Impossible Cloud|<host_logo>
```

Comment:
@laeccles I believe the shared logos point to 'Impossible Cloud' and 'Fluence'. Please share the logo for the other remaining host '1k(x)' as well.

Reply:
@WinsCodes Here you go!  
![E6LKHRv7_400x400](https://github.com/memser-spaceport/pln-events/assets/112510736/1bf982f0-6874-4856-a2ca-576ab993afc3)
cc: @pandeyanujk